frontend/config/agents.ts

```diff
@@ -89,10 +89,10 @@ export const AGENT_CONFIG: {
       },
     },
     requiresMasterSafesOn: [EvmChainId.Base],
-    serviceApi: MemeooorBaseService,
-    displayName: 'Agents.fun agent',
+    serviceApi: AgentsFunBaseService,
+    displayName: 'Agents.fun agent - Base',
     description:
-      'Autonomously post to Twitter, create and trade memecoins, and interact with other agents.',
+      'Autonomously posts to Twitter, create and trade memecoins, and interact with other agents. Agent is operating on Base chain.',
```

Comment:
why post -> posts, but no create -> creates, trade -> trades, interact -> interacts? I don't know english, but maybe the description had like a hidden (to) Autonomously post, ......